exonum/src/blockchain/service.rs

```diff
@@ -369,7 +383,7 @@ impl SharedNodeState {
             .map(|(c, e)| (*c, *e))
             .collect()
     }
-    /// Update internal state, from `Node` State`
+    /// Updates internal state, from `Node State`.
```

Comment:
IMO, `Node State` is incorrect formatting; suggest changing it to "`State` of a blockchain node".